suites/squid/common/regression/nvme-single-site-deploy-and-configure.yaml

```diff
@@ -0,0 +1,140 @@
+# Test suite deploy and configure single site cluster
+# global-conf: conf/squid/common/13node-4client-single-site-regression.yaml
+
+tests:
+  - test:
+      name: Setup pre-requisites
+      desc: Setup packages and configuration for cluster deployment
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Deploy cluster using cephadm
+      desc: Bootstrap and deploy services
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      config:
+        steps:
+          - config:
+              service: cephadm
+              command: bootstrap
+              args:
+                mon-ip: node1
+          - config:
+              service: host
+              command: add_hosts
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              service: osd
+              command: apply
+              args:
+                all-available-devices: true
+          - config:
+              command: shell
+              args:
+                - "ceph fs volume create cephfs"
+          - config:
+              service: mds
+              command: apply
+              args:
+                placement:
+                  label: mds
+              base_cmd_args:
+                verbose: true
+              pos_args:
+                - cephfs
+          - config:
+              command: shell
+              args:
+                - "ceph osd pool create rbd"
+          - config:
+              command: shell
+              args:
+                - "rbd pool init rbd"
+      abort-on-fail: true
+
+  - test:
+      name: Configure client
+      desc: Configure the RGW & RBD clients
+      module: test_client.py
+      config:
+        command: add
+        id: client.1
+        node: node18
+        install_packages:
+          - ceph-common
+          - rbd-nbd
+          - jq
+          - fio
```

Comment:
@sunilkumarn417 we can add nvme-cli package here too and think if we can initialize it. "modprobe nvme-fabrics"

Reply:
as this is not a requirement, leaving it.